packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -822,14 +822,26 @@ export const Dropdown: ComponentWithAs<'div', DropdownProps> &
     }
   };
 
+  const handleInputValueChange = (inputValue: string) => {
+    if (multiple) {
+      setStateAndInvokeHandler(['onSearchQueryChange'], null, {
+        searchQuery: '',
```

Comment:
This will not work. `searchQuery` is used to filter list of items, see `getFilteredValues()`. It should be cleared only on Esc and when an item is selected (`getSelectedItemAsString()`  in `Downshift.stateChangeTypes.clickItem` which you removed).

To repro, use keyboard to filter items in Search Multiple example.

It scares me that no test is failing :-/

Reply:
Thank you Miro for catching this. I made a change and added the filtering case for multiple search in unit test L1413